src/sql/workbench/api/node/extHostRequireInterceptor.ts

```diff
@@ -0,0 +1,85 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { TernarySearchTree } from 'vs/base/common/map';
+import { URI } from 'vs/base/common/uri';
+import { nullExtensionDescription } from 'vs/workbench/services/extensions/common/extensions';
+import { ExtensionIdentifier, IExtensionDescription } from 'vs/platform/extensions/common/extensions';
+import { INodeModuleFactory } from 'vs/workbench/api/node/extHostRequireInterceptor';
+import * as azdata from 'azdata';
+import * as sqlops from 'sqlops';
+import { IAzdataExtensionApiFactory, ISqlopsExtensionApiFactory } from 'sql/workbench/api/common/sqlExtHost.api.impl';
+
+export class AZDataNodeModuleFactory implements INodeModuleFactory {
```

Comment:
in the method above we had `Azdata` (in `createAzdataApiFactory`) here it's `AZData`.  I think we should try to be consistent with the casing for this common identifier.  I personally prefer `Azdata` though either is fine.